lib/analysis.js

```diff
@@ -4,12 +4,15 @@ var fs = require('fs');
 var walk = require('acorn/dist/walk');
 var plugin = require('./tern-def-api.js');
 var scope = require('./define-path-info.js');
+var logger = require('winston');
```

Comment:
I'd suggest to create separate 'logger.js' and move winston initialization there. Then you may be able to call
`var logger = require('./logger.js');`